tests/internal/scripts/ccuda/invalid_kernel_config.py

```diff
@@ -15,4 +15,4 @@ def opp(a, b):
     #invalid parameters
     opp[1.2, 7.0](a, b)
```

Comment:
this test should be done in the same way as the tests in `tests/cuda_test/test_kernel_semantic.py`
you can just move these to `tests/cuda_test/test_kernel_semantic.py` maybe having all the error test in one file will be easier to maintain.

Reply:
Fixed 7e71c89c30bf895c7d72a29fea81ae8e794b7fd6.